docs/_docs/configuration.md

```diff
@@ -564,6 +566,17 @@ defaults:
       layout: "specific-layout"
 ```
 
+<div class="note warning">
+  <h5>Globbing and Performance</h5>
+  <p>
+    Please note that globbing a path is a known performance-poison and is
```

Comment:
The `poison` sounds really terrifying! Can we rephrase?

> Please note that front matter defaults with glob paths have known performance problems.
> Windows users may notice especially high build times by adding globbing. We are tracking [in this issue](let's open an issue).

Reply:
> We are tracking [in this issue](let's open an issue).

I believe this is an issue with Ruby for Windows itself as Ruby is not optimized for Windows.. There's not much one can do here IMO..

So.. I'll just re-phrase the "poison" bit to something more mellow..